experimental/PropertyDDS/examples/property_inspector/README.md

```diff
@@ -10,7 +10,7 @@ Go back to the root folder and run:
 npm install
 alias fb='clear && node "$(git rev-parse --show-toplevel)/node_modules/.bin/fluid-build"'
 fb --install --symlink:full
-fb --all @fluid-experimental/partial-checkout  @fluid-experimental/property-query tinylicious
+fb --all @fluid-experimental/partial-inspector tinylicious
```

Comment:
shouldnt this be 
```suggestion
fb --all @fluid-experimental/property-inspector tinylicious
```

since  the[ package.json ](https://github.com/arno-123/FluidFramework/blob/eb008ac3135874bb0026e373ddcdacd477a81c7b/experimental/PropertyDDS/examples/property_inspector/package.json#L2) name

Reply:
Nice catch